index.html

```diff
@@ -89,6 +89,17 @@
                                     <p class="title is-size-5">
                                         Air Quality Metrics
                                     </p>
+
+                                    <p class="airInfo1"></p>
+                                    <p class="airInfo2"></p>
+                                    <p class="airInfo3"></p>
+                                    <p class="airInfo4"></p>
+                                    <p class="airInfo5"></p>
+                                    <p class="airInfo6"></p>
+                                    <p class="airInfo7"></p>
+                                    <p class="airInfo8"></p>
+                                    <!-- <img src="https://via.placeholder.com/300"> -->
```

Comment:
You can remove both <img> lines of code on 101 and 103, as they are no longer needed.